terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/ExistInCodeListTest.java

```diff
@@ -53,10 +55,20 @@ public class ExistInCodeListTest {
     @Inject
     CodeService codeService;
 
-    public ExistInCodeListTest() {
+    private Locale originalLocale;
+
+    @Before
+    public void setLocaleEnglish() {
+        ((LocalValidatorFactoryBean)validator).afterPropertiesSet();
+        originalLocale = Locale.getDefault();
         Locale.setDefault(Locale.ENGLISH);
     }
 
+    @After
+    public void setOriginalLocale() {
+        Locale.setDefault(originalLocale);
+    }
+
     @Before
     public void setUp() throws Exception {
     }
```

Comment:
- [ ] I believe that in Junit4, if multiple ``@Before`` existed, the order of loading was undefined.
This does not seem to be a problem because of the empty setUp, but other classes seem to describe processing in setUp, so please review the overall processing.

Reply:
88b8c8b
In one test class, the method to which `@BeforeClass/@AfterClass/@Before/@After` was given was modified to a maximum of one each.
If a method has no content, the entire method was deleted.
On the local machine
`mvn clean install -Dsurefire.runOrder=random`
10 times, and all 10 times `BUILD SUCCESS` was obtained, so I pushed it.
In GitLab environment, I ran mvn clean install -Dsurefire.runOrder=random 10 times and got `BUILD SUCCESS` all 10 times.